src/repair.c

```diff
@@ -105,8 +108,22 @@ static bool parse_opt(int opt, char *optarg)
 		cmdline_option_picky_whitelist = strdup_or_die(optarg);
 		return true;
 	case FLAG_EXTRA_FILES_ONLY:
-		cmdline_extra_files_only = optarg_to_bool(optarg);
+		cmdline_option_extra_files_only = optarg_to_bool(optarg);
 		return true;
+	case 'B': {
+		char *arg_copy = strdup_or_die(optarg);
+		char *token = strtok(arg_copy, ",");
```

Comment:
Don't duplicate code here. We can do it in a function. 
And I'm unsure about this implementation. In the case we set the bundles property in a config file it will be concatenated with the bundles from the command line. Is this expected?

Reply:
hmm didn't think of that, I don't think we should concatenate them , they should be overwritten

Reply:
nice catch by the way